python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -23,7 +23,13 @@
 from hypothesis import assume, given, settings
 from hypothesis.extra.pandas import column, data_frames, range_indexes
 
+import sys
+from tests.util.mark import ARCTICDB_USING_CONDA
 
+
+@pytest.mark.xfail(
+    ARCTICDB_USING_CONDA and sys.platform == "linux", reason="Test fails on linux conda, should be fixed by issue #1035"
+)
```

Comment:
Minor: might want to declare a wrapping `@xfail_on_linux_conda(reason=...)` function in `mark.py` so we can easily locate these.